### ISSUE TITLE
The first and subsequent times the application is started on Windows 7 it crashes.

### ISSUE THREAD

**Issue by virtuoushub** [MarginallyClever/Makelangelo#72]
tl;dr
Registry key: ```HKEY_CURRENT_USER\Software\JavaSoft\Prefs\/Language``` needs to contain
valid values for the "language" and "first time" keys or else the Makelangelo GUI will never start on Windows 7.
e.g.
```
"language"="/English"
"first time"="false"
```

## Findings
[MultilingualSupport#MultilingualSupport()](https://github.com/MarginallyClever/Makelangelo/blob/dev/java/src/Makelangelo/MultilingualSupport.java#L60) relies on [MainGUI#getSingleton()](https://github.com/MarginallyClever/Makelangelo/blob/dev/java/src/Makelangelo/MainGUI.java#L257-262) which [eventually](https://github.com/MarginallyClever/Makelangelo/blob/dev/java/src/Makelangelo/MainGUI.java#L185) relies back on [MultilingualSupport](https://github.com/MarginallyClever/Makelangelo/blob/dev/java/src/Makelangelo/MultilingualSupport.java). So depending on certain logic branches(e.g. the first time I loaded up the application on my Windows 7 box), the application will cause an infinite loop resulting in a ```java.lang.StackOverflowError```.

Judging by the comments for [MainGUI#CreateAndShowGUI()](https://github.com/MarginallyClever/Makelangelo/blob/dev/java/src/Makelangelo/MainGUI.java#L2252-2277), it seems like this is a relatively known issue.
```java
    // Create the GUI and show it.  For thread safety, this method should be invoked from the event-dispatching thread.
```

Unfortunately calling it from the event-dispatching thread does not insure thread safety in my case.

MultilingualSupport#ChooseLanguage() needs to be called after MainGUI and MultilingualSupport "singleton" objects are constructed. That or the logic for the "first time" preference needs to be altered.

In my branch I went with the former, and it worked just fine in my Windows 7 environment.
